src/components/templates/PartyMap/components/Map/MapRoom.scss

```diff
@@ -54,6 +54,7 @@ $label-visible: flex;
     display: $label-hidden;
     flex-direction: column;
     align-items: center;
+    font-family: "zoinks";
```

Comment:
It would be better if this was set using a variable as well.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1240/commits/4b4351f2e7c0d9b970954dc3303bfa8db9db3a37